include/modem/at_cmd.h

```diff
@@ -86,8 +84,7 @@ int at_cmd_init(void);
  * @retval -EIO is returned if the function failed to send the command.
  */
 int at_cmd_write_with_callback(const char *const cmd,
-					  at_cmd_handler_t  handler,
-					  enum at_cmd_state *state);
```

Comment:
The async API is not used yet but for a future version I would like to have a symmetric API. The `at_cmd_handler_t ` callback should be changed to a new one so the user could receive an error code and the result of the AT command response (state).
Would be nice to have this new API in the next version.

Reply:
Well noted